README.md

```diff
@@ -1 +1,38 @@
-# jitsi-meet-web-sdk
\ No newline at end of file
+# Jitsi Meet Web SDK
+The Jitsi Meet Web SDK provides the same user experience as the [Jitsi Meet](https://github.com/jitsi/jitsi-meet) app, in a customizable way which you can embed in your apps.
+
+## Install
+```bash
+npm install jitsi-meet-websdk
+```
+### Modules
+#### initExternal
+To import the Jitsi Meet External API in a non-React project:
+```js
+// index.js
+initExternal().then((externalApi) => { ... });
+
+// index.html
+window.onload = () => {
+  const api = new window.JitsiMeetExternalAPI("YOUR_DOMAIN", { roomName: "YOUR_CUSTOM_ROOM_NAME" });
```

Comment:
If the initExternal() function is used, then I think here the instantiation should be done on the result of that promise and not with windows.JitsiMeetExternalAPI